aptiche/lib/utils/string.dart

```diff
@@ -1,3 +1,8 @@
 //SVG Strings
+
 const String loginScreenSVG = 'assets/svgs/login_screen.svg';
 const String dataEntryScreen = 'assets/svgs/data_entry_screen.svg';
+const String homeScreenSVG = 'assets/svgs/TakeAQuiz.svg';
+
+//PNG Strings
+const String quizPNG = 'assets/images/quiz.png';
```

Comment:
image background fixed? shouldn't we ask for a better svg to AICHE?

Reply:
Yeah, they gave, I'll change it.